Dockerfile

```diff
@@ -1,17 +1,17 @@
 # => Build client
-FROM node:alpine as client_builder
+FROM node:16-alpine as client_builder
```

Comment:
Did `17` break things or `17.0.1`?

Reply:
When I tested this, node 17 was broken it gave some hash error. However, the other images weren't running due to a bash error:

Did you fix the `/bin/sh envsubst not found` issue? I was trying node images and was getting that error.

Reply:
I think 17 and 17.0.1 are identical according to Docker Hub.

The following tags are also all equivalent: 17, 17-bullseye, 17.0, 17.0-bullseye, 17.0.1, 17.0.1-bullseye, bullseye, current, current-bullseye, latest.

Reply:
Deploy it on heroku and see if it works

Reply:
Works

Reply:
ya, so 17 and latest all evaluate to 17.0.1. If 17.0.0 is fine, lets use that version.

Reply:
I think 17.0.0 is also broken.

Reply:
The issue is with version 17 and above, so we need to use 16. https://github.com/webpack/webpack/issues/14532

Reply:
Sounds good!